### DISCUSSION TITLE
How to do I rename my repo?

### DISCUSSION THREAD

**Discussion by quinnlovescode** [community/community#21562] [New to GitHub]
<p>I just created a new repo, and I made a typo and now I want to rename it. How do I do that?</p>

**Comment by a-a-ron** [comment, marked as answer]
<p>Hi <a class="mention" href="/quinnlovescode">@quinnlovescode</a>, this is a common enough question that GitHub has provided a help page to answer this question that you can find <a href="https://help.github.com/articles/renaming-a-repository/" rel="nofollow noopener">here.</a> There is more information provided in the help guide, but the steps are as follows;</p>
<ol>
<li>
<p>On GitHub, navigate to the main page of the repository.</p>
</li>
<li>
<p>Under your repository name, click <strong>Settings</strong>.</p>
</li>
<li>
<p>Under the <strong>Repository Name</strong> heading, type the new name of your repository. </p>
</li>
<li>
<p>Click <strong>Rename</strong>. </p>
</li>
</ol>

**Comment by quinnlovescode** [comment]
<p>Hi <a class="mention" href="/a-a-ron">@a-a-ron</a> Thank you so much. this is much easier than I thought it would be!</p>

**Comment by tarifalhasan** [comment]
when i was rename my github repository then i try to commit and update my repo face this error 
![image](https://user-images.githubusercontent.com/92237522/219609257-d9e2e839-1657-4891-a736-743ef6a6409a.png)

**Reply by fahimhasandev** [reply]
Try this --> git remote set-url origin <git@github.com:User/project-new.git>
